src/features/auth/PageContext.tsx

```diff
@@ -59,6 +62,13 @@ interface IPageContext {
     comment?: CommentView,
     comments?: CommentView[],
   ) => void;
+
+  presentAccountSwitcher: (
```

Comment:
Let's have this function take no arguments, and then add a function in PageContext to call presentAccountSwitcher with cssClass = small.

Reply:
Sounds good